tests/conftest.py

```diff
@@ -119,13 +122,15 @@ def comment_id(client, page_id) -> str:
     yield response["id"]
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module")
 def client(token: Optional[str]):
-    return Client({"auth": token})
+    with Client({"auth": token}) as client:
+        yield client
+    client.close()
 
 
 @pytest.fixture
 async def async_client(token: Optional[str]):
-    client = AsyncClient({"auth": token})
-    yield client
+    async with AsyncClient({"auth": token}) as client:
+        yield client
     await client.aclose()
```

Comment:
Why do we still need to manually close the clients if we now use the context manager? What's the point of using the context manager?

Reply:
I used the context manager to test both the "__enter__" and "__exit__" and the "close" method. Honestly I never used the client with a context manager so this is purely for coverage reasons. We can split the tests in two: one is using a classic approach and another one is using the context manager, for clarity?

Reply:
Ah! Well, ideally we'd want the context manager methods and close/aclose methods to have their own tests, but I guess we can just remove the close/aclose calls here and call it done for now. :)